app/views/admins/stats.mobile.haml

```diff
@@ -0,0 +1,56 @@
+.col-md-9
+  %h1= t(".usage_statistic")
+
+  %div
+    = form_tag("/admins/stats", method: "get", class: "form-inline") do
+      %select.form-control{name: "range"}
+        %option{value: "daily", selected: ("selected" if params[:range] == "daily")}
+          = t(".daily")
+        %option{value: "week", selected: ("selected" if params[:range] == "week")}
+          = t(".week")
+        %option{value: "2weeks", selected: ("selected" if params[:range] == "2weeks")}
+          = t(".2weeks")
+        %option{value: "month", selected: ("selected" if params[:range] == "month")}
+          = t(".month")
+
+      = submit_tag t(".go"), class: "btn btn-primary btn-block"
+
+  %h3
+    != t("admins.stats.display_results", segment: content_tag(:strong, @segment))
+
+  .row
+    - [:posts, :comments, :aspect_memberships, :users].each do |name|
+      - model = eval("@#{name.to_s}")
```

Comment:
Me randomly again, `eval` in this instance most likely isn't an actual security risk since you are limiting the use of it to your predefined array. Others may have a differing opinion to it though. There isn't really a clean fix for this besides manually writing out each case of posts, comments, etc. Which may be preferred as most ruby devs shy away from eval wherever possible.